src/main/java/supertracker/storage/TransactionStorage.java

```diff
@@ -147,6 +155,15 @@ public static void loadTransactionData() throws IOException {
         fileScanner.close();
     }
 
+    /**
+     * Takes string data from a line extracted from the data file and parses it to a Transaction object
+     *
+     * @param transactionData a String containing the data of a Transaction object's attributes
+     * @return a Transaction object parsed from the data string
+     * @throws Exception if the relevant attributes are unable to be extracted from the data string
+     * @throws TrackerException if the transaction date extracted from the string data is of a date that has not
+     * occurred yet
```

Comment:
Requires indentation

```suggestion
     *     occurred yet
```